YoCode/FeatureInfo/FeatureEvidence.cs

```diff
@@ -31,6 +33,11 @@ public void SetInconclusive(IEvidence reason)
             Evidence = reason;
         }
 
+        internal void GiveEvidence(string v)
+        {
+            throw new NotImplementedException();
```

Comment:
Is this deliberate? Why is it here?

Reply:
i think its something that was left over from another branch or after some merge with master